src/pages/PlayAndLearn/StartLearn.js

```diff
@@ -169,13 +170,34 @@ function StartLearn() {
             localStorage.setItem('contentText', tempContent[getitem].content[localStorage.getItem('apphomelang')].text);
           }
           scroll_to_top('smooth');
+          setNewTempContent(tempContent)
           set_load_cnt(load_cnt => Number(load_cnt + 1));
         }
       })
       .catch(err => console.log(err));
     }
 
   };
+  const [newTempContent, setNewTempContent] = useState([]); 
+
+  const handleChangeWord = () => {
+    console.log("Try New");
+    // Implement logic to select a new content
+    let getitem = content_id;
+      let old_getitem = getitem;
+      while (old_getitem === getitem) {
+        getitem = randomIntFromInterval(0, Number(newTempContent.length - 1));
+      }
+    localStorage.setItem('trysame', 'no');
+    localStorage.setItem('content_random_id', getitem);
```

Comment:
are we using this anywhere?

Reply:
yes we are using it in VoiceComapre.js line no 219